app/models/v2/rule_result.rb

```diff
@@ -43,14 +43,32 @@ class RuleResult < ApplicationRecord
 
     searchable_by :result, %i[eq ne in notin]
 
+    # Searching rule results by title should also consider rule result identifier label as it was before
     searchable_by :title, %i[eq neq like unlike] do |_key, op, val|
       val = "%#{val}%" if ['ILIKE', 'NOT ILIKE'].include?(op)
-      bind = ['IN', 'NOT IN'].include?(op) ? '(?)' : '?'
```

Comment:
I believe this has been added here by an accident. I don't remember any place where you were able to filter rules by multiple titles. API doc also doesn't mention it. I don't remember any FE place that uses this either so removed it.